packages/api/endpoints/dashboard.js

```diff
@@ -0,0 +1,28 @@
+'use strict';
+
+const router = require('express-promise-router')();
+const {
+  getS3Object,
+  getFileBucketAndKey
+} = require('@cumulus/common/aws');
+
+/**
+ * Given a path in the form of bucket/key, get the item from
+ * S3 and return it with the corret content type. This is used to server
+ * the dashboard from the specified bucket
+ *
+ * @param {Object} req - express request object
+ * @param {Object} res - express response object
+ * @returns {Promise<Object>} the promise of express response object
+ */
+async function get(req, res) {
+  const [bucket, key] = getFileBucketAndKey(req.params[0]);
+
+  const getObjectResponse = await getS3Object(bucket, key);
```

Comment:
I don't know if there would be any appreciable performance benefit, but I think you could pipe a stream of the S3 object into the `res` object

Reply:
Something like:

`return s3ObjectReadStream.pipe(res)`

Reply:
But I need to grab the content-type out too, how would I get that?

Reply:
Apparently you can do something like this:

```
s3.getObject({Bucket: myBucket, Key: myFile})
    .on('httpHeaders', function (code, headers) {
        if (code < 300) {
            res.set(_.pick(headers, 'content-type', 'content-length', 'last-modified'));
        }                            
    })
    .createReadStream()
    .on('error', next)
    .pipe(res);
```

See https://stackoverflow.com/questions/35782434/streaming-file-from-s3-with-express-including-information-on-length-and-filetype